docs/reference/api/usb/USBMIDI.md

```diff
@@ -29,3 +29,168 @@ int main() {
     }
 }
 ```
+
+### Play "Take Me Out to the Ball Game" example
```

Comment:
When the code goes onto master, we'll want to put this example in the teams repo. Until then, it's fine that it's hardcoded into the doc.